### PR TITLE
3795 store info

### PR DESCRIPTION
**Pull request by sergiomilici**

### CODE REVIEW

File: www/js/services/member_sql_service.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    };
+
+    //"qid TEXT," + // customer or manager account code (like NEW.AAA or NEW:AAA)
+    //"name TEXT," + // full name (of customer or manager)
+    //"company TEXT," + // company name, if any (for customer or manager)
+    //"place TEXT," + // customer location / manager's company account code  ??????
+    //"balance REAL," + // current balance (as of lastTx) / manager's rCard security code
+    //"rewards REAL," + // rewards to date (as of lastTx) / manager's permissions / photo ID (!rewards.matches(NUMERIC))
+    //"lastTx INTEGER," + // unixtime of last reconciled transaction / -1 for managers
+    //"photo BLOB);" // lo-res B&
```
Comment (jpdimond) [inline www/js/services/member_sql_service.js:18]: I'm glad you added a description of the table.

File: www/templates/menu.html [hunk 2]
```diff
@@ -23,12 +23,20 @@ <h1 class="title">{{ 'mainMenu' | translate }}</h1>
           {{ 'home' | translate }}
         </ion-item>
 
-        <ion-item menu-close ng-click="logout()" ng-show="!isCashierModeEnabled()">
+
+        <!-- Manager Mode && Cashier Mode Not Available of Prefs-->
+        <ion-item menu-close ng-click="logout()" ng-show="!isCashierModeAvailable() && !isCashierModeEnabled()">
```
Comment (jpdimond) [inline www/templates/menu.html:28]: Great.

File: www/js/services/sqlite_service.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+      return txPromise.promise;
+    };
+
+    SQLiteService.prototype.executeQuery_ = function(query, params) {
+      var txPromise = $q.defer();
+      var result;
+
+      this.db.transaction(function(tx) {
+        tx.executeSql(query, params, function(tx, res) {
+          console.log("executeSql OK: ", tx);
```
Comment (jpdimond) [inline www/js/services/sqlite_service.js]: We should clean up these consoles after we are done with the database stuff (probs not yet.)

File: test/js/services/sqlite_service_spec.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+      });
+    rootScope.$apply();
+  });
+
+
+  describe('Managing DB', function() {
+
+    it('Should create Members table', function(done) {
+
+      rootScope.$apply();
```
Comment (jpdimond) [inline test/js/services/sqlite_service_spec.js:59]: Is this test completed?

Comment (sergiomilici) [inline test/js/services/sqlite_service_spec.js:59]: No, i'm having a problem with sqlite when testing. Working on it

File: www/js/services/sqlite_service.js (hunk 3 above)
Comment (sergiomilici) [inline www/js/services/sqlite_service.js]: agreed

**Pull request merged**